src/python/datastore/data_types.py

```diff
@@ -1294,6 +1294,12 @@ def _pre_put_hook(self):
                        coverage_information_key(self.fuzzer, self.date))
 
 
+class CorpusTag(Model):
+  """Corpus Tags for use in cross-pollination."""
+  tag = StringProperty()
+  fuzz_target_name = StringProperty()
```

Comment:
fuzz_target_name is a bit ambiguous. It can mean the binary name, the project qualified name, or the fully qualified name.

Reply:
I am not certain which of these I want. What name is being used here in cross-pollination?https://github.com/google/clusterfuzz/blob/e93300847bbf483f8b44ec92f71fb10475007a6d/src/python/bot/tasks/corpus_pruning_task.py#L726

Reply:
This is the fully qualified name per the comment above https://github.com/google/clusterfuzz/blob/fcdea8165a7eafe80ccb9b2a77396ae3db11a3df/src/python/datastore/data_types.py#L1183

Ideally FuzzTargetJob should've named this properly too, but that's difficult to change now. Let's just name this properly in CorpusTag as fully_qualified_fuzz_target_name.

Reply:
Done. Thanks for the help! I didn't see that comment before.